envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,18 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    // [#not-implemented-hide:]
+    // Configures percentage of requests that will be routed to same zone
```

Comment:
This is misleading, as assuming this maps to the runtime value, this is just a feature flag which enables or disabled zone aware routing, not the exact % that will be routed to the same zone.

I would say: "Configured the percentage of requests that will be considered for zone aware routing if zone aware routing is configure."

Reply:
Question to make comment more clear : Is there some other flag to enable Zone Aware Routing ?

Reply:
The best thing we have is probably the FAQ entry here which you could link to? https://www.envoyproxy.io/docs/envoy/latest/faq/zone_aware_routing

Reply:
From here "https://github.com/envoyproxy/envoy/blob/master/source/common/upstream/load_balancer_impl.cc#L360" should it be "Zone aware routing is enabled if it is set to 100%"

Reply:
From here (https://github.com/envoyproxy/envoy/blob/master/source/common/upstream/load_balancer_impl.cc#L360) i see that percentage is not taken care of , am i right? In that case comment can be "Zone Aware Routing is enabled if set to 100%" . Thanks!

Reply:
@mattklein123  can we finalise on comment here ? Thanks!

Reply:
`featureEnabled` is a percentage based kill switch. You can either decide to support a percentage in the configuration (which would replace the existing default of 100 in the runtime call), or potentially just have a boolean in the config. I would probably keep a percentage as you have it.